src/index.ts

```diff
@@ -152,10 +155,10 @@ function getHandlers(
       // call onSwipeStart if present and is first swipe event
       eventData.first && props.onSwipeStart && props.onSwipeStart(eventData);
 
-      // Call onSwiping if present
+      // call onSwiping if present
       props.onSwiping && props.onSwiping(eventData);
 
-      // track if a swipe is cancelable(handler for swiping or swiped(dir) exists)
+      // track if a swipe is cancelable (handler for swiping or swiped(dir) exists)
       // so we can call preventDefault if needed
       let cancelablePageSwipe = false;
       if (props.onSwiping || props.onSwiped || `onSwiped${dir}` in props) {
```

Comment:
```suggestion
      if (props.onSwiping || props.onSwiped || props[`onSwiped${dir}`]) {
```
📓 NOTE to @hartzis fix this and add test for `{ onSwipedLeft: undefined }`

Reply:
- [x] added fix! and documented in changelog